static/app/components/events/interfaces/performance/resources.tsx

```diff
@@ -37,6 +48,11 @@ const LinkSection = styled('div')`
   grid-template-columns: 1fr 1fr;
   grid-row-gap: ${space(1)};
 
+  @media (max-width: ${p => p.theme.breakpoints.small}) {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
I think you can keep it as a grid and just change it to one column right?

Reply:
True, I guess it makes more sense to keep it as a grid 👍